api/migration/update

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+. /usr/libexec/nethserver/api/lib/helper_functions
+
+set_migration_status() {
+    app_id=$1
+    status=$2
+    ns8_leader=$(/sbin/e-smith/config getprop ns8 Host)
+    /usr/bin/mkdir -p /var/lib/nethserver/agent/${ns8_leader}
```

Comment:
Is it necessary to create a state directory for each cluster?
I'd prefer to create/reset always the same path in the login/logut script. What do you think?

Reply:
I'm afraid multiple and partial migrations are hard to manage, also for the sysadmin. 

Furthermore 
- a module can still be in ns7 or migrated elsewhere. Once migrated we must remember its state and switching the state dir implies forget its state.
- ns7 cannot be disconnected from ns8 after account provider has been migrated.